x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps/compute_steps.tsx

```diff
@@ -273,6 +273,7 @@ export const ManagedSteps: React.FunctionComponent<InstructionProps> = ({
           selectedApiKeyId,
           isK8s,
           cloudSecurityIntegration,
+          fleetServerHost: fleetServerHosts?.[0],
```

Comment:
fleetServerHosts contains the fleet server host for the policy. We pick the first one like this in the install commands too: https://github.com/hop-dev/kibana/blob/c66f58ae0cbbb1c35a8f023f5507e60aff94dda4/x-pack/plugins/fleet/public/components/enrollment_instructions/manual/index.tsx#L23